app/client/config/webpack.config.js

```diff
@@ -183,7 +166,7 @@ module.exports = function (webpackEnv) {
         {
           loader: require.resolve("resolve-url-loader"),
           options: {
-            sourceMap: isEnvProduction ? shouldUseSourceMap : isEnvDevelopment,
+            sourceMap: isEnvProduction ? true : isEnvDevelopment,
```

Comment:
I think it might just be like that.

```suggestion
            sourceMap: isEnvProduction,
```